ext/dcos-installer/dcos_installer/action_lib/__init__.py

```diff
@@ -41,6 +41,21 @@ def run_preflight(config, pf_script_path='/genconf/serve/dcos_install.sh', block
     pf = get_async_runner(config, targets, async_delegate=async_delegate)
     chains = []
 
+    binary_chain = ssh.utils.CommandChain('preflight_compatibility')
+
+    # for Ubuntu/Debian derivatives
+    binary_chain.add_execute(
```

Comment:
I agree. Breaking out a preflight for every distro or even just Debian/Ubuntu derivatives seems like it would get a little messy.